addons/website_slides/controllers/main.py

```diff
@@ -240,7 +240,7 @@ def slides_channel_home(self, **post):
         domain = request.website.website_domain()
         channels_all = request.env['slide.channel'].search(domain)
         if not request.env.user._is_public():
-            channels_my = channels_all.filtered(lambda channel: channel.is_member).sorted('completion', reverse=True)[:3]
+            channels_my = channels_all.filtered(lambda channel: channel.is_member).sorted(lambda channel: not channel.completed and -channel.completion)[:3]
```

Comment:
whut ?  `-channel.completion` Typo ? real shady weird magic stuff ?

Reply:
`channels_all.filtered(lambda channel: channel.is_member).sorted(lambda channel: 100 if channel.completed else channel.completion, reverse=True)[:3]`

@awa-odoo : Thanks ! ;)

Reply:
If a course is completed, we don't want to see it in first position but in last, so I replace 100 by 0

channels_my = channels_all.filtered(lambda channel: channel.is_member).sorted(lambda channel: 0 if channel.completed else channel.completion, reverse=True)[:3]